website/addons/wiki/static/spellcheck_ace.js

```diff
@@ -0,0 +1,102 @@
+// You also need to load in typo.js and jquery.js
```

Comment:
It's fine to have these 3rd party modules in our repo since they're not on npm or bower, but any 3rd-party code should go in `static/vendor/`. Wouldn't hurt to add a folder for ace plugins.